optimizely/event/event_factory.py

```diff
@@ -89,10 +89,20 @@ def _create_visitor(cls, event, logger):
     """
 
         if isinstance(event, user_event.ImpressionEvent):
-            metadata = payload.Metadata(event.flag_key, event.rule_key, event.rule_type, event.variation.key)
-            decision = payload.Decision(event.experiment.layerId, event.experiment.id, event.variation.id, metadata)
+            experiment_layerId, experiment_id, variation_id, variation_key = '', '', '', ''
```

Comment:
`experiment_layer_id` or `experiment_layerid`

Reply:
yeah, I thought about it, but we already have  event.experiment.layerId